testing/testcafe/tests/editors/radioGroup/common.ts

```diff
@@ -64,10 +67,30 @@ test('Radio buttons placed into the template should not be selected after clicki
   await checkGroup(firstChildGroup, true);
   await checkGroup(secondChildGroup, false, true);
   await checkGroup(thirdChildGroup, false, false, true);
-}).before(async () => createWidget('dxRadioGroup', {
-  items: [{}, {}, {}],
-  itemTemplate: () => ($('<div>') as any).dxRadioGroup({
-    dataSource: [{}, {}, {}],
-    layout: 'horizontal',
-  }),
-}));
+}).before(async () => {
+  await appendElementTo('#container', 'div', 'template');
```

Comment:
Why we should change this test code and add additional element?

Reply:
fixed